config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```
@@ -279,3 +279,31 @@ presubmits:
           limits:
             cpu: "2"
             memory: "6Gi"
+  - name: pull-kueue-test-scalability-main
```

```suggestion
  - name: pull-kueue-test-scheduling-perf-main
```

Given that we want to change the name of the directives #1997

---

done